test.py

```diff
@@ -100,6 +108,8 @@ def run_test(test_args: "tuple[pathlib.Path, str, pathlib.Path]", args):
 				test_addrs[symbol.entry["st_value"]] = symbol.name
 
 	fname = (results_path / "config" / test_file.stem).with_suffix(".ini")
+	if args.debug_jit:
```

Comment:
I would say this is not a problem to be solved here, LLVMJIT is rarely used anyways.

Reply:
I can drop the assertion warning, but we need a way to disable the jit_debug because otherwise it wouldn’t be possible to run the tests against the LLVMJIT.

Reply:
does LLVMJIT complain when this option is set? from what I see here https://github.com/tum-ei-eda/etiss/blob/b24bc484d1498d6cdc4587a4dbd246adce0e5f26/JITImpl/LLVM/src/LLVMJIT.cpp#L219 debug should be supported just fine...